data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,856 @@ mission "Coalition License Patch"
+			`	He explains that he used to run an orphanage here, but has now gotten too old for the job. He has arranged for the four kids to be transferred to a foster home on <destination>.`
+			`	"Take care of them, the Saryds there will," he says, patting the head of the youngest and shortest of the kids. "Retiring, I am, but abandon them, I will not. An apartment close to the foster home, I will purchase."`
+			`	He offers to pay you <payment> to bring the five of them and their <cargo> to <destination>.`
+			choice
+				`	"Of course I'll help. Here, let me help you get the luggage into my ship."`
+					accept
+				`	"I hope things work out for you, but I'm actually not going to stay in or fly around Coalition space for long now."`
+					decline
+	on complete
+		"coalition jobs" += 3
+		payment 268000
+		conversation
+			`You help the old Arach caretaker and some workers take the luggage out of your ship then head to the orphanage.`
+			`	A Saryd couple runs the place and greets you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child and some Kimek too.`
+			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`
+
+
+
+mission "Arachi Cartoonist 1"
+	minor
+	name "A New Design"
+	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
+	to offer
+		has "Coalition: First Contact: done"
+	source "Mebla's Portion"
+	destination "Prime"
+	on offer
+		conversation
+			`As you're moving along one of the larger underground tunnels that connect the spaceport here, a dozen makeshift stands, each with a different apparent 'theme' to them, captures your attention. When you get closer, it's clear that the event is something of a convention, each stand managed by a different artist that lives here.`
+			`	After you're done looking at a stand that showcased various fantasy attires and move on to the neighboring one, it's owner jumps as soon as he turns to see you, looking dumbfounded. He introduces himself as Drugub, a cartoonist of local fame for his science fiction comics, showing various examples of them as he speaks with you. "Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "That you came here, lucky I am. To expand my creative horizon, I need, and seeing the general shape of some human ships would."`
+			choice
+				`	"I can help with that. Do you have any type of ship in mind?"`
+				`	"Sorry, but I won't be heading back to human space for a while."`
```

Comment:
I'd change this second choice to not mention going to human space, as it isn't immediately apparent to me that that is what Drugub is asking. (It could be that he just wants to see my ship.)

Reply:
changed it up